lib/tty.js

```diff
@@ -38,7 +38,7 @@ function ReadStream(fd, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(fd, options);
   if (fd >> 0 !== fd || fd < 0)
-    throw new RangeError('fd must be positive integer: ' + fd);
+    throw new errors.RangeError('ERR_INVALID_FD', fd);
```

Comment:
You're passing the `fd` in as an argument but you're not using it in the error message.